neo/rawio/openephysrawio.py

```diff
@@ -87,13 +88,12 @@ def _parse_header(self):
             all_first_timestamps = []
             all_last_timestamps = []
             all_samplerate = []
-            for continuous_filename in info['continuous'][seg_index]:
+            for chan_id, continuous_filename in enumerate(info['continuous'][seg_index]):
                 fullname = os.path.join(self.dirname, continuous_filename)
                 chan_info = read_file_header(fullname)
 
                 s = continuous_filename.replace('.continuous', '').split('_')
                 processor_id, ch_name = s[0], s[1]
-                chan_id = int(ch_name.replace('CH', ''))
```

Comment:
I don't anderstand this.

In neo rawio, the chan_id is something that match the original file format. Sometimes is is one based, sometimes can contains gaps.

Here, with enumerate it is more a chan_index.

In rawio the chan_index is the position, inside the "sig_channels" vector.

Reply:
Could we go back to chan_id even if chan_id can in some case have multiple entries.
 ```
            chan_str = re.split(r'(\d+)', s[1])[0]
            chan_id = int(ch_name.replace(chan_str, ''))
 ```
Here it is a channel index which is redundant to the channel position.